### PR TITLE
contacts cross

### PR DESCRIPTION
**Pull request by PogudoTanya**

### CODE REVIEW

File: src/screens/ChooseContact/components/ContactServerList/ContactServerList.jsx [hunk 1]
```diff
@@ -32,18 +38,18 @@ function ContactServerList({ title, listPhoneContacts, navigation, sendFlow }) {
...
+      {listCrossContacts.map(({ givenName, username }, index) => (
         <Contact
-          isLastContact={index === listPhoneContacts.length - 1}
+          isLastContact={index === listCrossContacts.length - 1}
           icon={getInitials(givenName)}
           onContactClick={() => {
             handleContactClick(givenName);
           }}
-          key={recordID}
+          key={Math.random()}
```
Comment (liubaterehova) [inline src/screens/ChooseContact/components/ContactServerList/ContactServerList.jsx:48]: key={recepientId}

**Review by liubaterehova — CHANGES_REQUESTED** [review]

File: src/screens/ChooseContact/ChooseContact.jsx [hunk 2]
```diff
@@ -10,53 +10,75 @@ import PropTypes from 'prop-types';
...
 import MainHeader from 'components/MainHeader';
 import DismissKeyboard from 'components/DismissKeyboard';
 import NoContact from 'components/NoContact';
 import ContactServerList from './components/ContactServerList';
 
 import styles from './ChooseContact.styles';
 
+const getListPhoneContacts = (contacts) => {
+  return contacts
+    .filter(({ phoneNumbers }) => {
```
Comment (apsevkin) [inline src/screens/ChooseContact/ChooseContact.jsx:23]: let's use arrow functions without body

File: src/screens/ChooseContact/ChooseContact.jsx [hunk 3]
```diff
@@ -10,53 +10,75 @@ import PropTypes from 'prop-types';
...
+
 function ChooseContact({ navigation, route }) {
   const { sendFlow } = route.params;
+
   const [listPhoneContacts, setListPhoneContacts] = useState([]);
+  const [listCrossContacts, setListCrossContacts] = useState([]);
 
-  useEffect(() => {
+  const getContactsFromPhoneBook = useCallback(() => {
     if (Platform.OS === 'android') {
```
Comment (apsevkin) [inline src/screens/ChooseContact/ChooseContact.jsx:38]: use async function

File: src/screens/ChooseContact/ChooseContact.jsx [hunk 4]
```diff
@@ -10,53 +10,75 @@ import PropTypes from 'prop-types';
...
+    })
+    .map(({ givenName, phoneNumbers }) => {
+      return { givenName, phoneNumber: phoneNumbers[0].number };
+    });
+};
+
 function ChooseContact({ navigation, route }) {
   const { sendFlow } = route.params;
+
   const [listPhoneContacts, setListPhoneContacts] = useState([]);
```
Comment (apsevkin) [inline src/screens/ChooseContact/ChooseContact.jsx:34]: get rid of this state

**Review by apsevkin — CHANGES_REQUESTED** [review]

**Review by apsevkin — APPROVED** [review]

**Review by liubaterehova — APPROVED** [review]

**Pull request merged**